mmv1/products/compute/api.yaml

```diff
@@ -13911,6 +13911,35 @@ objects:
                       description: |
                         CIDR IP address range.
                       item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
Apologies for the confusion, but adding these fields does nothing for the provider. compute_security_policy is handwritten for Terraform, and this yaml entry existed when MM still generated Ansible/Inspec downstreams. If you [see here](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/compute/terraform.yaml#L2650), the api.yaml entry is actually excluded from Terraform generation. It was not reasonable for you to know that, and I've created an issue to track removing all obsolete api.yaml entries to avoid more confusion in the future. https://github.com/hashicorp/terraform-provider-google/issues/12509

For now, I think we should remove this chunk from your changes, just to avoid confusing anyone looking at git history.

Reply:
No problem at all! I figured this config was not doing anything, but just wasn't sure if we were trying to keep it up-to-date for some future state of the tool. Agreed that if we don't want them to be updated, it's probably best to remove them entirely.

I'll remove this entry from the PR, thanks!